src/box/iproto.cc

```diff
@@ -356,6 +356,15 @@ struct iproto_msg
 	 * ibuf object.
 	 */
 	size_t len;
+	/**
+	 * Pointer to the start of unparsed request stored in @a p_ibuf.
+	 * It is used to dump request to flight recorder (if available) in
+	 * TX thread. It is guaranteed that @a reqstart points to the valid
+	 * position: rpos of input buffer is moved after processing the message;
+	 * meanwhile requests are handled in the order they are stored in
+	 * the buffer.
+	 */
+	char *reqstart;
```

Comment:
Nit: Please make it const. The first argument of `flightrec_write_request` should be const as well.

Reply:
I've also added commit which adds extra box include dirs to build main.cc (according to the review of flight recorder).